src/components/index.ts

```diff
@@ -1,12 +1,12 @@
 export { default as ActivityChart } from './ActivityChart.vue'
```

Comment:
Changes here are for new components and I ran a sort on the list which changed the order of a few items